sites/footers/_base.scss

```diff
@@ -18,34 +24,120 @@
 
 	.brand {
 		background: #f8f8f8;
-		line-height: 30px;
+		padding: 1.5em 0 3em;
 
 		object,
 		img {
 			height: 40px;
-			margin-bottom: 2em;
-			margin-top: 20px;
 			width: auto;
 		}
+
+		.brand-row {
+			align-items: center;
+			display: flex;
+
+			.ftr-urlt-lnk {
+				flex: 1 1 auto;
+				padding: 0;
+
+				ul {
+					list-style-type: none;
+					margin: 0;
+					padding: 0;
+
+					li {
+						margin-bottom: 0;
+					}
+				}
+			}
+		}
+
+		/* START: To support footer <= v2.0 */
+		.row {
+			ul {
+				list-style-type: none;
+				margin-bottom: 0;
+				padding: 0;
+				padding-top: 1em;
+
+				li {
+					display: inline-block;
+					margin-right: .5em;
+
+					[dir=rtl] & {
+						margin-left: .5em;
+						margin-right: 0;
+					}
+
+					&:not(:first-child):before {
+						content: "\2022";
+						margin-right: .75em;
+
+						[dir=rtl] & {
+							margin-left: .75em;
+							margin-right: 0;
+						}
+					}
+				}
+			}
+
+			.ftr-urlt-lnk {
+				padding-bottom: 0;
+				padding-top: 0;
+			}
+		}
+		/* END */
 	}
 
-	.ftr-urlt-lnk {
-		ul {
-			margin: 0;
-			padding: 0;
+	.wtrmrk {
+		align-self: flex-end;
+	}
+
+	.gc-contextual {
+		background-color: $bg-gcthemelight;
+
+		nav {
+			padding-bottom: 0;
 		}
 
-		li {
-			&:before {
-				content: "\2022";
-				margin-right: $footer-about-li-horizontal-spacing;
```

Comment:
Delete completely the SASS variable which seem to be very specific `$footer-about-li-horizontal-spacing`